assets/section-main-product.css

```diff
@@ -1366,14 +1366,71 @@ a.product__text {
 /* Product-thumbnail snippet */
 
 .product-media-container {
+  --aspect-ratio: var(--preview-ratio);
+  --ratio-percent: calc(1 / var(--aspect-ratio) * 100%);
+  position: relative;
   width: 100%;
   max-width: calc(100% - calc(var(--media-border-width) * 2));
 }
 
+.product-media-container.constrain-height {
+  --viewport-offset: 400px;
```

Comment:
I was thinking actually that because this is a value we chose "randomly" that we could leave a comment at least just here to mention that. 
So that if we ever need to revisit it, we don't need to go anywhere else to look for answers. 

Something like: `amount chosen and decided after general testing`

Reply:
Just thought it'd be good thing to keep doing going forward. We've found ourselves sometimes not being sure how some approaches/values were decided. 
That's the only thing left I'd like to see in the PR yeah 🤔 👍 Not a huge thing but I feel like let's start doing it now.

Reply:
https://github.com/Shopify/dawn/pull/2052/commits/54bd15dd8edca2ae329c570de608e56e26e631d6

Reply:
Much appreciated 🤗